breakpoint|../core/throw_ret01.v3 4|0
stop|stopOnBreakpoint
bt|throw_ret01.main|../core/throw_ret01.v3|4
variable|0|this|throw_ret01|throw_ret01|false
variable|1|a|0|int|false
stop|end
result|!DivideByZeroException
	in throw_ret01.main() [../core/throw_ret01.v3 @ 4:26]

